go/consensus/tendermint/apps/scheduler/genesis.go

```diff
@@ -143,10 +135,17 @@ func (app *schedulerApplication) InitChain(ctx *abciAPI.Context, req types.Reque
 	//
 	// Sort of stupid it needs to be done this way, but tendermint doesn't
 	// appear to pass ABCI the validator set anywhere other than InitChain.
-	if err := state.PutCurrentValidators(ctx, currentValidators); err != nil {
+	if err = state.PutCurrentValidators(ctx, currentValidators); err != nil {
 		return fmt.Errorf("failed to set validator set: %w", err)
 	}
 
+	if !doc.Scheduler.Parameters.DebugBypassStake {
```

Comment:
This check should (also) be performed in the genesis document sanity check so it fails early.

Reply:
oh yeah, we should